examples/nix-shell/shell.ncl

```diff
@@ -0,0 +1,20 @@
+{
+  inputs = {
+    hello = { input = "nixpkgs" }
+  }
+
+  , output = fun { pkgs, nix, .. } =>
+    { # (optional) name of the shell
```

Comment:
It’s a bit strange to have one (slightly misindented) comment for this field and none for the others. Maybe we could comment all or none of them.

Also, shouldn’t we use annotations for this kind of stuff?

Reply:
I agree with the strange indentation.

I think those are more like comments to explain the example further, but not documentation that is meant to be queried (in fact, `nix.ncl` may already define a documentation for those fields, or is likely to do so in the future).